tutorials/v7/ntuple/ntpl015_processor_join.C

```diff
@@ -0,0 +1,112 @@
+/// \file
+/// \ingroup tutorial_ntuple
+/// \notebook
+/// Demonstrate the RNTupleProcessor for joining RNTuples
+///
+/// \macro_image
+/// \macro_code
+///
+/// \date November 2024
+/// \author The ROOT Team
```

Comment:
I would say your name is better suited here

Reply:
All other tutorials also have "The ROOT team", in this case I would prefer to keep this consistency.